iot-hub/Quickstarts/simulated-device-2/Program.cs

```diff
@@ -0,0 +1,147 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// This application uses the Azure IoT Hub device SDK for .NET
+// For samples see: https://github.com/Azure/azure-iot-sdk-csharp/tree/master/iothub/device/samples
+
+using Microsoft.Azure.Devices.Client;
+using System;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SimulatedDevice
+{
+    /// <summary>
+    /// This sample illustrates the very basics of a device app sending telemetry and receiving a command.
+    /// For a more comprehensive device app sample, please see
+    /// <see href="https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/iot-hub/Samples/device/DeviceReconnectionSample"/>.
+    /// </summary>
+    internal class Program
+    {
+        private static DeviceClient s_deviceClient;
+
+        // The device connection string to authenticate the device with your IoT hub.
+        // Using the Azure CLI:
+        // az iot hub device-identity show-connection-string --hub-name {YourIoTHubName} --device-id MyDotnetDevice --output table
+        private static string s_connectionString = "{Your device connection string here}";
+
+        private static TimeSpan s_telemetryInterval = TimeSpan.FromSeconds(1); // Seconds
+
+        private static async Task Main(string[] args)
+        {
+            Console.WriteLine("IoT Hub Quickstarts #1 - Simulated device.");
+
+            // This sample accepts the device connection string as a parameter, if present
+            CheckForConnectionStringArgument(args);
+
+            // Connect to the IoT hub using the MQTT protocol
+            s_deviceClient = DeviceClient.CreateFromConnectionString(s_connectionString, TransportType.Mqtt);
```

Comment:
suggestion: We can call out the TransportType as a private static field so it's easier to find/change by the users.